crazyflie_py/crazyflie_py/crazyflie.py

```diff
@@ -976,3 +997,16 @@ def cmdFullState(self, pos, vel, acc, yaw, omega):
         self.cmdFullStateMsg.twist.angular.y = omega[1]
         self.cmdFullStateMsg.twist.angular.z = omega[2]
         self.cmdFullStatePublisher.publish(self.cmdFullStateMsg)
+
+    def get_statuses(self):
+        '''
+        Obtain a list containing the status of each crazyflie controlled by the Crazyserver
+        Each status is a dict, see status_topic_callback() for more details
+        '''
+        
+        # self.get_logger().info(f'Crazyserver.get_statuses() was called')
+        statuses = []
+        for cf in self.crazyflies:
+            statuses.append(cf.get_status())
+            
+        return statuses
```

Comment:
I think it's better to remove this and call the status() function in infinite_flight.py, e.g., `allcfs.crazyflies[0].status()`.